### ISSUE TITLE
v4: Error in dev mode on hot reload

### ISSUE THREAD

**Issue by johnf** [marklawlor/nativewind#757]
**Describe the bug**
When I change a className on a component in a particular way I get a crash with the following error.

`ERROR  TypeError: Cannot assign to read-only property 'validated'`

This is happening at https://github.com/facebook/react/blob/main/packages/react/src/ReactElementValidator.js#L182

**To Reproduce**
It's a bit tricky to reproduce. My theory at the moment is that it happens when you change an existing style to one that isn't in the CSS file.

1. Create a component with a class of top-10
2. Change the class to top-20

**Expected behavior**
Stlye should update without a crash

**Expo Snack**
Are there v4 snacks yet?

**Screenshots**

![image](https://github.com/marklawlor/nativewind/assets/42590/db7aef09-9019-4f3a-a007-d968ede7f521)

**Additional context**

I think this is just some dev mode validation. Commenting out the line in React solves the issue and doesn't seem to have any impact, which has been my workaround.

I'm currently on 4.0.23 but this has been an issue for a while, just haven't had the time to file it

**Comment by wojciechkrol** [comment]
The same issue occurs here. I have a condition set for the background, and when I toggle it, I encounter the same error:

```jsx
          <View
            className={`
              transition duration-500
              flex py-5 px-10 dark:bg-green-500
              ${toggle ? 'bg-orange-500' : 'bg-teal-500'},
            `}>
            <Feather name="chevron-left" size={16} color="#FFF" />
          </View>
```
![image](https://github.com/marklawlor/nativewind/assets/14108143/c988a886-6fd7-4955-b82a-e9b1daee5f78)
---
NativeWind: 4.0.23

**Comment by tiimmmoooo** [comment]
I got the same issue when hot-reloading and changing my classNames / style / vars.
![IMG_5046](https://github.com/marklawlor/nativewind/assets/24858736/cc1e4831-238a-446d-a6f3-b8b5d09e72f7)

**Comment by ck-euan** [comment]
Yeah same thing is happening for me, it's almost unusable at this point as it errors on almost every save

**Comment by marklawlor** [comment]
Fixed in `4.0.28`. I believe the issue was hot reloading animated components. If your still having the issue can you please create a new issue with a full reproduction
